flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaTableTestBase.java

```diff
@@ -135,8 +134,16 @@ public void createTestTopic(String topic, int numPartitions, int replicationFact
         properties.put(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG, getBootstrapServers());
         try (AdminClient admin = AdminClient.create(properties)) {
             admin.createTopics(
-                    Collections.singletonList(
-                            new NewTopic(topic, numPartitions, (short) replicationFactor)));
+                            Collections.singletonList(
+                                    new NewTopic(topic, numPartitions, (short) replicationFactor)))
+                    .all()
+                    .get();
+        } catch (Exception e) {
+            throw new IllegalStateException(
```

Comment:
We could have come up with a general exception for these kind of cases as well since they all reflect the same issue. That way, we could have encapsulated the `String.format` and varargs in the constructor. Just as an idea... I'm not insisting on that change considering that we're planning to create a follow-up where all of that is revisited, anyway.

Reply:
I would leave it in the beautification phase. I think it would be good to fix a 1+ year issue. Pretty sure several hundred CI and dev hours we can spare by letting beautification for later :)